superset/db_engine_specs/base.py

```diff
@@ -1097,3 +1112,26 @@ def get_extra_params(database: "Database") -> Dict[str, Any]:
     def is_readonly_query(cls, parsed_query: ParsedQuery) -> bool:
         """Pessimistic readonly, 100% sure statement won't mutate anything"""
         return parsed_query.is_select() or parsed_query.is_explain()
+
+    def get_column_spec(
+        self,
+        column_name: Optional[str],
+        native_type: str,
+        source: utils.ColumnTypeSource = utils.ColumnTypeSource.GET_TABLE,
+    ) -> utils.ColumnSpec:
+
+        column_type, generic_type = self.get_sqla_column_type(native_type)
+        is_dttm = generic_type == utils.GenericDataType.TEMPORAL
+
+        if column_name:  # Further logic to be implemented
+            pass
+        if (
+            source == utils.ColumnTypeSource.CURSOR_DESCRIPION
+        ):  # Further logic to be implemented
+            pass
```

Comment:
For now we can assume `GET_TABLE` and `CURSOR_DESCRIPTION` is handled with the same matching logic - this can later be refined for engines where this makes a difference. Also, we might consider keeping the base implementation as simple as possible, and leaving the more complex logic to the individual db engine specs.